src/components/UserPlaylists/component.js

```diff
@@ -1,16 +1,16 @@
-import React, { Component } from "react";
-import PropTypes from "prop-types";
-import "./UserPlaylists.css";
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './UserPlaylists.css';
 
 class UserPlaylists extends Component {
   componentWillReceiveProps(nextProps) {
-    if (nextProps.userId !== "" && nextProps.token !== "") {
+    if (nextProps.userId !== '' && nextProps.token !== '') {
       this.props.fetchPlaylistsMenu(nextProps.userId, nextProps.token);
     }
   }
 
   renderPlaylists() {
-    return this.props.playlistMenu.map(playlist => {
+    return this.props.playlistMenu.map((playlist) => {
```

Comment:
Better to check if the array will be present or not.
`return this.props.playlistMenu && this.props.playlistMenu.map((playlist) => {`